### ISSUE TITLE
Publish with `--watch` flag hangs

### ISSUE THREAD

**Issue by bgins** [fission-suite/fission#531]
# Summary

## Problem

`fission app publish --watch` hangs.

### Impact

Users cannot use the `--watch` flag to watch a project and publish on changes.

## Solution

Not known.

# Detail

**Describe the bug**

When running the `fission app publish --watch` command, the CLI gets stuck. In verbose mode, the last few outputs are:

```
2021-06-21 11:39:26.147822: [debug] ⚙️ 🆔 Loaded Server DID: did:key:z6MkgYGF3thn8k1Fv4p4dWXKtsXCnLH7q9yw4QgNPULDmDKB
2021-06-21 11:39:26.147948: [debug] 🧬🔬 Parsing signing key
2021-06-21 11:39:26.148011: [info] 🧬👍 Loaded signing key
2021-06-21 11:39:26.148068: [user] ✈️  Pushing to remote
```

**To Reproduce**
1. Run `fission app publish --watch --verbose` in the directory of a registered app
2. Watch logs until they hang.

**Expected behavior**

The `fission app publish --watch` should publish and return to a watching state.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - CLI Version: 2.13.0.0

**Comment by walkah** [comment]
Is this what was just fixed by #530 ? or is this something else?

**Comment by bgins** [comment]
Ah yes, it is! I missed #530. Closing this one.
